### DISCUSSION TITLE
Kafka Connect Cluster High Memory Usage

### DISCUSSION THREAD

**Discussion by mozarik** [strimzi/strimzi-kafka-operator#8851] [General]
<img width="680" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/39885486/92c2c3d9-de02-4355-8836-2b8cc548e12f">

When its starting it just in 600. But when I leave it on its increasing to 6000 and sometimes have OOMKilled 

Here is the history in my grafana 
<img width="622" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/39885486/4345999f-79cf-4cf6-9371-5f9443173ad3">

**Comment by scholzj** [comment]
I guess for a start you should share your configurations. In general, there is nothing wrong with Connect using 6GB of memory when it is configured to do so or when it needs it.
